<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>No Roll Setup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Joint Chain Parent</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="jointChainLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="jointChainPushButton">
           <property name="text">
            <string>&lt;&lt;&lt; Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearShldrJointPushButton_2">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Shoulder Joint</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="shldrJointLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="loadShldrJointPushButton">
           <property name="text">
            <string>&lt;&lt;&lt; Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearShldrJointPushButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Elbow Joint</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="elbowJointLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="loadElbowJointPushButton">
           <property name="text">
            <string>&lt;&lt;&lt; Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearElbowJointpushButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Wrist Joint</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="wristJointLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="loadWristJointPushButton">
           <property name="text">
            <string>&lt;&lt;&lt; Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearWristJointpushButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Number of joints for upper arm</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="sliderPosition">
             <number>6</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="clavJointLcdNumber">
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="numDigits">
             <number>2</number>
            </property>
            <property name="mode">
             <enum>QLCDNumber::Dec</enum>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="value" stdset="0">
             <double>6.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Number of joints for forearm</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QSlider" name="horizontalSlider_2">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="sliderPosition">
             <number>6</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="elbowJointLcdNumber">
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="numDigits">
             <number>2</number>
            </property>
            <property name="mode">
             <enum>QLCDNumber::Dec</enum>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="value" stdset="0">
             <double>6.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="createPushButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>clavJointLcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>elbowJointLcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearShldrJointPushButton</sender>
   <signal>clicked()</signal>
   <receiver>shldrJointLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearElbowJointpushButton</sender>
   <signal>clicked()</signal>
   <receiver>elbowJointLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearWristJointpushButton</sender>
   <signal>clicked()</signal>
   <receiver>wristJointLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearShldrJointPushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>jointChainLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
